src/scripts/test_cli.py

```diff
@@ -1928,6 +1928,7 @@ def main(args=None):
 
         for result in results:
             result.get()
+        pool.close()
```

Comment:
I believe, `TreadPool` implements a "context manager". I.e. you could use this along those lines:

```python
with ThreadPool() as pool:
   # ...

Reply:
Done